libraries/types/dag_block/include/dag/dag_block.hpp

```diff
@@ -20,7 +21,9 @@ class DagBlock {
   level_t level_ = 0;
   vec_blk_t tips_;
   vec_trx_t trxs_;  // transactions
+  estimations_vec_t trx_estimations_;
```

Comment:
I dont think I would automatically know that trx_estimation is trxs_gas_estimations if I was not reviewing this PR